RodSoriano/nekupu
# Added PWA setup

## Description

- 🐛 Updated the NestJS start script to resolve issues with environment variable handling during initialization.
- ✨ Added Progressive Web App (PWA) support to the frontend for better offline capabilities and improved performance.
- 🎨 Applied consistent formatting to the API code for better readability and maintainability.

## Type of Change

Please check the type of change you are making:

- [x] Enhancement (modification of existing functionality)

## How to Test

1. Run the application using the root `dev` script.
2. Verify that the environment variables are properly loaded in the backend.
3. Check the frontend for PWA support (e.g., service worker registration, manifest).
4. Ensure code formatting in the API remains consistent with the project's coding standards.

## Verification

*Aspects of the code to be reviewed:*

- [x] Code quality and structure
- [ ] Security vulnerabilities
- [ ] Completeness of documentation

## Suggested Reviewers

- @RodSoriano

## Labels

- `enhancement`
